### ISSUE TITLE
Triaging missing assets found in the error logs

### ISSUE THREAD

**Issue by mbland** [18F/web-design-standards#747]
I grepped through the error log for yesterday and today via:

```sh
$ grep '^2015/09/2[89] .*designstandards' error.log > 2015-09-29-designstandards-error-log.txt
```

And found the assets listed below were getting errors. The number at the beginning is the number of error occurrences. Not all of the errors are due to errors in the site itself, but those with a larger number of occurrences probably are. For example, I did notice that the `assets-styleguide/img` errors referred to files in `assets/img`, e.g. correct8.{png,svg} and correct9.svg, and correct9.png doesn't exist at all.

```sh
$ grep 'open() .*designstandards/assets.* failed' 2015-09-29-designstandards-error-log.txt | cut -d' ' -f7 | sed -e 's#.*designstandards/\([^"]*\).*#\1#' | sort | uniq -c > 2015-09-29-missing-files.txt

$ cat 2015-09-29-missing-files.txt

      2 assets/css/google-fonts.css
     14 assets/css/google-fonts.css../fonts/merriweather-bold-webfont.eot
      7 assets/css/google-fonts.css../fonts/merriweather-bold-webfont.ttf
      7 assets/css/google-fonts.css../fonts/merriweather-bold-webfont.woff
      7 assets/css/google-fonts.css../fonts/merriweather-bold-webfont.woff2
     14 assets/css/google-fonts.css../fonts/merriweather-italic-webfont.eot
      7 assets/css/google-fonts.css../fonts/merriweather-italic-webfont.ttf
      7 assets/css/google-fonts.css../fonts/merriweather-italic-webfont.woff
      7 assets/css/google-fonts.css../fonts/merriweather-italic-webfont.woff2
     14 assets/css/google-fonts.css../fonts/merriweather-light-webfont.eot
      7 assets/css/google-fonts.css../fonts/merriweather-light-webfont.ttf
      7 assets/css/google-fonts.css../fonts/merriweather-light-webfont.woff
      7 assets/css/google-fonts.css../fonts/merriweather-light-webfont.woff2
     14 assets/css/google-fonts.css../fonts/merriweather-regular-webfont.eot
      6 assets/css/google-fonts.css../fonts/merriweather-regular-webfont.ttf
      7 assets/css/google-fonts.css../fonts/merriweather-regular-webfont.woff
      7 assets/css/google-fonts.css../fonts/merriweather-regular-webfont.woff2
     14 assets/css/google-fonts.css../fonts/sourcesanspro-bold-webfont.eot
      7 assets/css/google-fonts.css../fonts/sourcesanspro-bold-webfont.ttf
      7 assets/css/google-fonts.css../fonts/sourcesanspro-bold-webfont.woff
      7 assets/css/google-fonts.css../fonts/sourcesanspro-bold-webfont.woff2
     14 assets/css/google-fonts.css../fonts/sourcesanspro-italic-webfont.eot
      7 assets/css/google-fonts.css../fonts/sourcesanspro-italic-webfont.ttf
      6 assets/css/google-fonts.css../fonts/sourcesanspro-italic-webfont.woff
      7 assets/css/google-fonts.css../fonts/sourcesanspro-italic-webfont.woff2
     14 assets/css/google-fonts.css../fonts/sourcesanspro-light-webfont.eot
      7 assets/css/google-fonts.css../fonts/sourcesanspro-light-webfont.ttf
      7 assets/css/google-fonts.css../fonts/sourcesanspro-light-webfont.woff
      6 assets/css/google-fonts.css../fonts/sourcesanspro-light-webfont.woff2
     14 assets/css/google-fonts.css../fonts/sourcesanspro-regular-webfont.eot
      7 assets/css/google-fonts.css../fonts/sourcesanspro-regular-webfont.ttf
      7 assets/css/google-fonts.css../fonts/sourcesanspro-regular-webfont.woff
      6 assets/css/google-fonts.css../fonts/sourcesanspro-regular-webfont.woff2
      7 assets/fonts/merriweather-bold-webfont.svg#source_sans_probold
      7 assets/fonts/merriweather-italic-webfont.svg#source_sans_proitalic
      7 assets/fonts/merriweather-light-webfont.svg#source_sans_proregular
      7 assets/fonts/merriweather-regular-webfont.svg#source_sans_proitalic
      7 assets/fonts/sourcesanspro-bold-webfont.svg#source_sans_probold
      7 assets/fonts/sourcesanspro-italic-webfont.svg#source_sans_proitalic
      7 assets/fonts/sourcesanspro-light-webfont.svg#source_sans_prolight
      7 assets/fonts/sourcesanspro-regular-webfont.svg#source_sans_proregular
      1 assets/img/correct1.svg
      1 assets/img/correct2.svg
     10 assets/img/correct9.png
      2 assets/img/favicons/favicon-144.png
      2 assets/img/favicons/favicon-192.png
      2 assets/img/us_flag_large.png
      1 assets/img/us_flag_lge.png
      1 assets/img/us_flag.png
     10 assets/img/us_flag_small.png
      4 assets/js/vendor/jquery-1.11.3.min.js
      2 assets-styleguide/css/homepage.css
      1 assets-styleguide/i..
    130 assets-styleguide/img/alerts/error.png
    131 assets-styleguide/img/alerts/error.svg
    128 assets-styleguide/img/alerts/info.png
    131 assets-styleguide/img/alerts/info.svg
    132 assets-styleguide/img/alerts/success.png
    130 assets-styleguide/img/alerts/success.svg
    130 assets-styleguide/img/alerts/warning.png
    130 assets-styleguide/img/alerts/warning.svg
    136 assets-styleguide/img/arrow-down.png
    221 assets-styleguide/img/arrow-down.svg
    138 assets-styleguide/img/arrow-right.png
    136 assets-styleguide/img/arrow-right.svg
    135 assets-styleguide/img/correct8.png
    142 assets-styleguide/img/correct8.svg
    137 assets-styleguide/img/correct9.png
    133 assets-styleguide/img/correct9.svg
      2 assets-styleguide/img/default_example_emanifest.png
      4 assets-styleguide/img/home/example_playbook.jpg
      4 assets-styleguide/img/home/example_VAappeals_mock.png
      1 assets-styleguide/img/home/example_VAappealsmock.png
      4 assets-styleguide/img/home/example_voteUSAgov.jpg
    827 assets-styleguide/img/home/example_voteUSAgov.png
      3 assets-styleguide/img/home/homepage_illustrations_508_box_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_508_box.png
      4 assets-styleguide/img/home/homepage_illustrations_best_easiest_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_best_easiest.png
      4 assets-styleguide/img/home/homepage_illustrations_designer_2x.png
     26 assets-styleguide/img/home/homepage_illustrations_designer.png
      3 assets-styleguide/img/home/homepage_illustrations_developer_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_developer.png
      4 assets-styleguide/img/home/homepage_illustrations_flexible_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_flexible.png
      4 assets-styleguide/img/home/homepage_illustrations_reuse_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_reuse.png
      4 assets-styleguide/img/home/homepage_illustrations_ui_components_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_ui_components.png
      4 assets-styleguide/img/home/homepage_illustrations_visual_style_guide_2x.png
      2 assets-styleguide/img/home/homepage_illustrations_visual_style_guide.png
      2 assets-styleguide/img/merriweatheronly_example_playbook.png
    129 assets-styleguide/img/minus.png
    127 assets-styleguide/img/minus.svg
    130 assets-styleguide/img/plus.png
    131 assets-styleguide/img/plus.svg
      2 assets-styleguide/img/robust_example_standardshome.png
    136 assets-styleguide/img/search.png
    135 assets-styleguide/img/search.svg
      2 assets-styleguide/img/ssponly_example_va.png
      2 assets-styleguide/js/styleguide.js
      2 assets-styleguide/js/vendor/prism.js
```

**Comment by maya** [comment]
@mbland what action do you want us to take?

**Comment by mbland** [comment]
I figured y'all might check out how those asset references are ending up in `assets-styleguide` and figure out how to properly reference them or remove the references entirely. I'd imagine the higher-occurrence errors would take priority. (Just noticed that `assets-styleguide/img/home/example_voteUSAgov.png` showed up 827 times!)

I also didn't know if any of those other missing assets were red flags of any kind, hence I included them all. And I wonder if some of the errors are due to access on devices that weren't explicitly tested, e.g. a media query specific to a particular phone+browser combination resulted in an asset reference that didn't exist.
